flutter/lib/src/default_integrations.dart

```diff
@@ -350,6 +349,7 @@ class LoadReleaseIntegration extends Integration<SentryFlutterOptions> {
   @override
   FutureOr<void> call(Hub hub, SentryFlutterOptions options) async {
     try {
+      // For non Flutter apps, we read the environment variables in sentry_dart
```

Comment:
this comment is off IMO, `kIsWeb` maybe be true for a non flutter app, I'd prefer `non Android or iOS` or something more specific